pypeit/scripts/install_telluric.py

```diff
@@ -42,12 +41,9 @@ def main(args):
             raise ValueError(f'No telluric grid files found in {args.path}')
 
         # Get the directory for the grids and make sure it exists
-        grid_dir = resource_filename('pypeit', 'data/telluric/atm_grids/')
-        if not os.path.isdir(grid_dir):
-            raise NotADirectoryError(f'Unable to find {grid_dir}.  Check your installation.')
+        if not os.path.isdir(data.Paths.telgrid):
+            raise NotADirectoryError(f'Unable to find {data.Paths.telgrid}.  Check your installation.')
```

Comment:
Same thought about putting this in the `Paths` class.

Reply:
See comment related to `install_ql_masters.py`.